### ISSUE TITLE
Enable toggle-all

### ISSUE THREAD

**Issue by sepuckett86** [orbitjs/todomvc-ember-orbit#6]
Toggle-all input icon appears only after editing a todo, and when then icon is clicked, this error message is logged:

```
todo-list.js:87 Uncaught TypeError: this.args.todos is not iterable
    at _class.toggleAll (todo-list.js:87)
```

**Comment by dgeb** [comment]
Thanks for reporting!

**Comment by dgeb** [comment]
This should be (finally) fixed via #16 and the latest ember-orbit beta. Thanks again for reporting!
